subsys/mgmt/mcumgr/lib/cmd/img_mgmt/src/img_mgmt.c

```diff
@@ -257,24 +257,34 @@ img_mgmt_erase(struct mgmt_ctxt *ctxt)
 {
 	struct image_version ver;
 	int rc;
+	zcbor_state_t *zsd = ctxt->cnbd->zs;
 	zcbor_state_t *zse = ctxt->cnbe->zs;
 	bool ok;
+	uint32_t slot = 1;
+	size_t decoded = 0;
+
+	struct zcbor_map_decode_key_val image_erase_decode[] = {
+		ZCBOR_MAP_DECODE_KEY_VAL(slot, zcbor_uint32_decode, &slot),
+	};
+
+	ok = zcbor_map_decode_bulk(zsd, image_erase_decode,
```

Comment:
`ok` is not used anywhere.
Should we even continue if this fails?

Reply:
Fixed